src/lib/pages/past-txs/index.tsx

```diff
@@ -159,7 +160,9 @@ const PastTxs = () => {
               setCurrentPage(1);
               setValue("search", e.target.value);
             }}
-            placeholder="Search with Transaction Hash or Contract Address"
+            placeholder={`Search with Transaction Hash${
+              wasm.enabled ? " or Contract Address" : ""
+            }`}
```

Comment:
How about move feature?

Reply:
add later when after deciding what we can search